### DISCUSSION TITLE
How long does it take to complete the stitching process once all tiles are downloaded?

### DISCUSSION THREAD

**Discussion by d351r3d** [domlysz/BlenderGIS#556] [Q&A] [answered]
I downloaded 88k tiles and my computer sat for hours. I wanted to know how long should it take to finalize the stitching of the images together before it creates a plane and the copy pastes the stitched together image on it? I waited like 2+ hours and it didn't complete.

How would I check to make sure that it is doing what it should be doing? I realize there's a console but I'm new to Blender and only using it for a basic purpose. BlenderGIS

**Comment by domlysz** [comment, marked as answer]
Stitching should be relatively fast, I think in your case it just fails because the images you trying to build is too large. Zoom 23 as you say in #555 is a very high resolution, at this scale one pixel represents 25cm. The stitching process require putting all tiles in memory so at this scale you can easily freezing the system with a too large area. Even if the stitching process complete, I don't think Blender will be able to handle such a wide texture because it's not designed for. So the real question is  : do you really need this level of detail for your area of interest ? Keep in mind : rendering a large area will not necessarily benefit of a very detailed texture, and Blender is not a GIS system designed to handle large geographic dataset. So as a now advertised user your job is to find the best balance between your area of interest and the level of detail required for your project, but neither the addon nor blender will give you both a very large area and an high level of detail.

**Comment by domlysz** [comment]
88000 tiles of 256px at  0.00025km per pixel represents a square of 2816 * 2816 km, this is really too ambitious, this image will require  1 516 765 Go to fit in memory !

**Reply by d351r3d** [reply]
ok. I figured that was the case. That's really rough. Thanks for the response though. I'll try the same area but with a lower zoom. Probably zoom 20 amd get back to you. The thing is the quality helps with knowing what's in that area because I was planning on having people build the area up in blender or another 3d application based on what  Street View has available since it shows how the building looks. The quality of the map would would help with more stores being shown since it'll make it easier.

thanks again for the reply. I definitely appreciate it.

**Reply by blendermagic** [reply]
In my experience, image tiles at higher zoom levels aren’t necessarily of a higher quality, because they are simply resampled, as evidenced by the fuzziness that you see. They look fine when looking down from a great height, but oblique close-ups are deadly. You will be better off using procedural textures instead in that case, because **no** zoom level will cure the problem. Another is files that end up being gigabytes in size!
